util/util.go

```diff
@@ -83,11 +87,35 @@ func SaveWebauthnSession(session *sessions.Session, key string, sessionData *web
 	return nil
 }
 
+func RandInit() {
```

Comment:
this method may be able to be cleaned up, however leaving this in place _incase_ I missed anywhere.

Reply:
Removed within #[f7f4776](https://github.com/Quiq/webauthn_proxy/pull/3/commits/f7f47768feb76d64a577beefdc78b2e580702616)